CHANGELOG.md

```diff
@@ -13,6 +18,12 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 - Daemon/stats: more resilient cpu sampling [moby/moby#36519](https://github.com/moby/moby/pull/36519)
 * Containerd: update to 1.0.3 release [moby/moby#36749](https://github.com/moby/moby/pull/36749)
 - Fix Windows layer leak when write fails [moby/moby#36728](https://github.com/moby/moby/pull/36728)
+* Don't make container mount unbindable [moby/moby#36768](https://github.com/moby/moby/pull/36768)
+* Ensure hijacked connection implements a CloseWrite fucntion [moby/moby#36517](https://github.com/moby/moby/pull/36517) 
```

Comment:
typo: fucntion; also this wasn't merged yet, and still had some discussion, so may be good to hold off until it's fully landed; https://github.com/docker/docker-ce/pull/467